images/manageiq-base/container-assets/container_env

```diff
@@ -4,45 +4,44 @@
 
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 
-if [[ ! -f /var/www/miq/vmdb/config/database.yml ]]; then
-  [[ -f /run/secrets/postgresql/POSTGRESQL_HOSTNAME ]] && database_hostname_file=$(cat /run/secrets/postgresql/POSTGRESQL_HOSTNAME)
-  [[ -f /run/secrets/postgresql/POSTGRESQL_PASSWORD ]] && database_password_file=$(cat /run/secrets/postgresql/POSTGRESQL_PASSWORD)
-  [[ -f /run/secrets/postgresql/POSTGRESQL_PORT ]] && database_port_file=$(cat /run/secrets/postgresql/POSTGRESQL_PORT)
-  [[ -f /run/secrets/postgresql/POSTGRESQL_USER ]] && database_username_file=$(cat /run/secrets/postgresql/POSTGRESQL_USER)
-  database_hostname=${DATABASE_HOSTNAME:-$database_hostname_file}
-  database_password=${DATABASE_PASSWORD:-$database_password_file}
-  database_port=${DATABASE_PORT:-$database_port_file}
-  DATABASE_SSL_MODE=${DATABASE_SSL_MODE:-prefer}
-  database_username=${DATABASE_USERNAME:-$database_username_file}
+[[ -f /run/secrets/postgresql/POSTGRESQL_HOSTNAME ]] && database_hostname_file=$(cat /run/secrets/postgresql/POSTGRESQL_HOSTNAME)
+[[ -f /run/secrets/postgresql/POSTGRESQL_PASSWORD ]] && database_password_file=$(cat /run/secrets/postgresql/POSTGRESQL_PASSWORD)
+[[ -f /run/secrets/postgresql/POSTGRESQL_PORT ]] && database_port_file=$(cat /run/secrets/postgresql/POSTGRESQL_PORT)
+[[ -f /run/secrets/postgresql/POSTGRESQL_USER ]] && database_username_file=$(cat /run/secrets/postgresql/POSTGRESQL_USER)
+database_hostname=${DATABASE_HOSTNAME:-$database_hostname_file}
+database_hostname=${database_hostname:-localhost}
+database_password=${DATABASE_PASSWORD:-$database_password_file}
+database_port=${DATABASE_PORT:-$database_port_file}
+database_port=${database_port:-5432}
+DATABASE_SSL_MODE=${DATABASE_SSL_MODE:-prefer}
+database_username=${DATABASE_USERNAME:-$database_username_file}
+database_name=${DATABASE_NAME:-vmdb_production}
```

Comment:
Minor but previously I used a non-existent db as the default.  This was so that I wouldn't accidentally connect to any real db.  The db doesn't need to exist to start a rails console, which was the use case for the defaults.  I guess I'm ok either way, but what do you think @bdunne

Reply:
Done